File: app/src/main/java/com/pluu/webtoon/adapter/viewholder/DaumRIghtViewHolder.java
package com.pluu.webtoon.adapter.viewholder;

import android.content.Context;
import android.view.View;
import android.widget.ImageView;
import android.widget.TextView;

import com.pluu.webtoon.R;
import com.pluu.webtoon.item.ChatView;

import butterknife.BindView;
import butterknife.ButterKnife;

/**
 * 우측 표시용 Item ViewHolder
 * Created by PLUUSYSTEM-SURFACE on 2016-05-21.
 */
public class DaumRIghtViewHolder extends BaseChattingViewHolder {

    @BindView(R.id.rightProfileImageView)
    ImageView rightProfileImageView;
    @BindView(R.id.rightNameTextView)
    TextView rightNameTextView;
    @BindView(R.id.rightMessageTextView)
    TextView rightMessageTextView;

    public DaumRIghtViewHolder(View v) {
        super(v);
        ButterKnife.bind(this, v);
    }

    @Override
    public void bind(Context context, ChatView item) {
        loadProfileImage(context, rightProfileImageView, item.getImgUrl());
        rightNameTextView.setText(item.getName());
        rightMessageTextView.setText(item.getText());
    }
}

File: app/src/main/java/com/pluu/webtoon/adapter/viewholder/DaumRIghtViewHolder.kt
package com.pluu.webtoon.adapter.viewholder

import android.content.Context
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import com.pluu.webtoon.R
import com.pluu.webtoon.item.ChatView
import kotlinx.android.synthetic.main.view_chatting_right_layout.view.*

/**
 * 우측 표시용 Item ViewHolder
 * Created by  pluu on 2017-05-02.
 */
class DaumRIghtViewHolder(v: View) : BaseChattingViewHolder(v) {

    override fun bind(context: Context, item: ChatView?) {
        if (item == null) return

        item.imgUrl?.let {
            loadProfileImage(context, itemView.rightProfileImageView, it)
        }
        itemView.rightNameTextView.text = item.name
        itemView.rightMessageTextView.text = item.text
    }

    companion object {

        fun newInstance(parent: ViewGroup) = DaumRIghtViewHolder(
                LayoutInflater.from(parent.context)
                        .inflate(R.layout.view_chatting_right_layout, parent, false))

    }
}
